components/src/dynamo/global_planner/scale_handler.py

```
@@ -257,31 +569,200 @@ async def scale_request(self, request: ScaleRequest):
+                                await partner_connector.set_component_replicas(
+                                    [
+                                        TargetReplica(
+                                            sub_component_type=SubComponentType(
+                                                partner_sub
+                                            ),
+                                            desired_replicas=partner_desired,
+                                        )
+                                    ],
+                                    blocking=request.blocking,
+                                )
+                            except Exception as pair_err:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)